lib/Runtime/Debug/BreakpointProbe.cpp

```diff
@@ -12,6 +12,7 @@ namespace Js
         characterOffset(statement.statement.begin),
         byteOffset(statement.bytecodeSpan.begin)
     {
+        this->breakPointId = functionBody->GetScriptContext()->GetThreadContext()->GetDebugManager()->GetNextBreakpointId();
```

Comment:
but it is loosing an abstraction -- need to depends on the calling DebugManager.

---
In reply to: [62775405](https://github.com/Microsoft/ChakraCore/pull/926#discussion_r62775405) [](ancestors = 62775405,62774317)